_security-plugin/configuration/ldap.md

```diff
@@ -426,12 +426,15 @@ If you don't use or have a role subtree, you can disable the role search complet
 rolesearch_enabled: false
 ```
 
+## Advanced settings
 
-### (Advanced) Control LDAP user attributes
+The advanced settings presented below are optional for an essential LDAP configuration. They can, however, improve efficiency and performance for the LDAP implementation. 
```

Comment:
These settings also improve security as well. Perhaps, `efficiency, performance and security`?